src/Pages/yoonseonghwang/Login/Login.js

```diff
@@ -1,8 +1,157 @@
 import React from 'react';
+import './Login.scss';
 
 class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      backgroundColor: '#c0dffd',
+      disabled: true,
+      id: '',
+      pw: '',
+    };
+  }
+
+  handleIdInput = event => {
+    this.setState(
+      {
+        id: event.target.value,
+      },
+      this.eventHandler
+    );
+  };
+
+  handlePwInput = event => {
+    this.setState(
+      {
+        pw: event.target.value,
+      },
+      this.eventHandler
+    );
+  };
+
+  eventHandler = () => {
+    !this.state.id.includes('@') || this.state.pw.length < 5
+      ? this.setState({ disabled: true, backgroundColor: '#c0dffd' })
+      : this.setState({ disabled: false, backgroundColor: '#0095f6' });
+  };
```

Comment:
```suggestion
  eventHandler = () => {
    const { id, pw } = this.state
    const isValid = !id.includes('@') || pw.length < 5
    
    this.setState({
      disabled: isValid,
      backGroundColor: isValid ? '#c0dffd' : '#0095f6'
    })
  };
```
- 결국 만들어주고 싶은 객체의 형태가 있고 그 경우 한번에 대하여 setState를 하는 것이기 때문에 위와 같이 적용해보면 어떨까요?
- 여기서 조금더 맞는 방향으로 바꿔보자면 disabled만 state로 관리하고, backgroundColor는 클래스명으로 정의해두고 disabled의 상태에 따라 변하도록 해주는게 좋아보입니다